src/CapabilityService.WebApi/Program.cs

```diff
@@ -8,19 +8,33 @@
 using Serilog;
 using Serilog.Events;
 using Serilog.Formatting.Compact;
+using Serilog.Sinks.SystemConsole.Themes;
 
 namespace DFDS.CapabilityService.WebApi
 {
     public class Program
     {
         public static int Main(string[] args)
         {
-            Log.Logger = new LoggerConfiguration()
+            bool.TryParse(Environment.GetEnvironmentVariable("CAPABILITY_SERVICE_HUMAN_LOG"), out var humanLog);
+
+            var logcfg = new LoggerConfiguration()
                 .MinimumLevel.Information()
                 .MinimumLevel.Override("Microsoft", LogEventLevel.Warning)
-                .Enrich.FromLogContext()
-                .WriteTo.Console(new CompactJsonFormatter())
-                .CreateLogger();
+                .MinimumLevel.Override(source: "Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker", LogEventLevel.Information)
```

Comment:
Build error line 24 and 25: Named arguments must appear after all fixed arguments.

Reply:
Hrm, it builds just fine here. Which version are you on?

Reply:
Using rider or _dotnet build_?

Version info:
_dotnet --info_
.NET Core SDK (reflecting any global.json):
 Version:   2.2.105
 Commit:    7cecb35b92

Runtime Environment:
 OS Name:     manjaro
 OS Version:  
 OS Platform: Linux
 RID:         arch-x64
 Base Path:   /opt/dotnet/sdk/2.2.105/

Host (useful for support):
  Version: 2.2.3
  Commit:  6b8ad509b6

.NET Core SDKs installed:
  2.1.402 [/opt/dotnet/sdk]
  2.1.403 [/opt/dotnet/sdk]
  2.1.500 [/opt/dotnet/sdk]
  2.2.105 [/opt/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.4 [/opt/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.5 [/opt/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.6 [/opt/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.4 [/opt/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.5 [/opt/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.6 [/opt/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.4 [/opt/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.5 [/opt/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.6 [/opt/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.3 [/opt/dotnet/shared/Microsoft.NETCore.App]

Reply:
Adding an explicit C# version to the .csproj allows for compilation, e.g.
```
  <PropertyGroup>
    <TargetFramework>netcoreapp2.2</TargetFramework>
    <RootNamespace>DFDS.CapabilityService.WebApi</RootNamespace>
    <AssemblyName>CapabilityService.WebApi</AssemblyName>
    <LangVersion>7.2</LangVersion>
  </PropertyGroup>
```

Reply:
Nice find. Still not sure why though.